gcampus/core/models/index/base.py

```diff
@@ -0,0 +1,108 @@
+#  Copyright (C) 2021-2022 desklab gUG (haftungsbeschränkt)
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Affero General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Affero General Public License for more details.
+#
+#  You should have received a copy of the GNU Affero General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from __future__ import annotations
+
+__ALL__ = ["WaterQualityIndex"]
+
+from typing import Union
+
+from django.contrib.gis.db import models
```

Comment:
I think you meant to use

```python
from django.db import models
```

`django.contrib.gis.db.models` is a superset of `django.db.models` so it does not cause any issues but `django.db.models` is more fitting here.